samples/dnn/text_detection.py

```diff
@@ -2,24 +2,62 @@
 import cv2 as cv
 import math
 import argparse
+import numpy as np
 
 ############ Add argument parser for command line arguments ############
-parser = argparse.ArgumentParser(description='Use this script to run TensorFlow implementation (https://github.com/argman/EAST) of EAST: An Efficient and Accurate Scene Text Detector (https://arxiv.org/abs/1704.03155v2)')
-parser.add_argument('--input', help='Path to input image or video file. Skip this argument to capture frames from a camera.')
-parser.add_argument('--model', required=True,
-                    help='Path to a binary .pb file of model contains trained weights.')
+parser = argparse.ArgumentParser(description="Use this script to run TensorFlow implementation (https://github.com/argman/EAST) of "
+                                             "EAST: An Efficient and Accurate Scene Text Detector (https://arxiv.org/abs/1704.03155v2)")
+parser.add_argument('--input',
+                    help='Path to input image or video file. Skip this argument to capture frames from a camera.')
+parser.add_argument('--model', '-m', required=True,
+                    help='Path to a binary .pb file contains trained detector network.')
+parser.add_argument('--ocr', required=True, default="crnn.onnx",
+                    help="Path to a binary .pb or .onnx file contains trained recognition network", )
 parser.add_argument('--width', type=int, default=320,
                     help='Preprocess input image by resizing to a specific width. It should be multiple by 32.')
-parser.add_argument('--height',type=int, default=320,
+parser.add_argument('--height', type=int, default=320,
                     help='Preprocess input image by resizing to a specific height. It should be multiple by 32.')
-parser.add_argument('--thr',type=float, default=0.5,
+parser.add_argument('--thr', type=float, default=0.5,
                     help='Confidence threshold.')
-parser.add_argument('--nms',type=float, default=0.4,
+parser.add_argument('--nms', type=float, default=0.4,
                     help='Non-maximum suppression threshold.')
 args = parser.parse_args()
 
+
 ############ Utility functions ############
-def decode(scores, geometry, scoreThresh):
+
+def fourPointsTransform(frame, vertices):
+    outputSize = (32, 100)
+    targetVertices = np.array([
+        [0, outputSize[0] - 1],
+        [0, 0],
+        [outputSize[1] - 1, 0],
+        [outputSize[1] - 1, outputSize[0] - 1]], dtype="float32")
+
+    rotationMatrix = cv.getPerspectiveTransform(vertices, targetVertices)
+    result = cv.warpPerspective(frame, rotationMatrix, (outputSize[1], outputSize[0]))
```

Comment:
Why not to have `outputSize = (100, 32)`? So we can pass it directly into the method.